src/ThemeChanger/index.js

```diff
@@ -22,9 +24,7 @@ const ThemeChanger = () => {
     <ApplicationConsumer>
       {({ darkMode }) => (
         <div>
-          <button onClick={handleChange}>
-            {darkMode ? "Light Mode" : "Dark Mode"}
-          </button>
+          <Icon icon={chart_1_2} size={22} onClick={handleChange} style={{cursor:'pointer'}} />
```

Comment:
Please style using scss modules, so make ThemeChanger.module.scss inside the same folder and import

Reply:
Ok 😃. Initially I thought of doing that, but wasn't sure whether it was better to create a file for just one css change. I will do that now.